src/main/java/frc/robot/Robot.java

```diff
@@ -519,6 +519,7 @@ private void executeIntakeStateMachine() {
 
         // Check transition to shooting after storage of ball
         checkTransitionToShooting();
+        checkTransitionToClimbing();
```

Comment:
Should only check transition to climbing if:
- Intake state machine is in IDLE or READY_TO_SHOOT
- Shooting state machine is in IDLE

Reply:
Fixed